tests/Responses/Expected/mugs_list_page.json

```diff
@@ -234,7 +234,7 @@
                     "default": false,
                     "urlParameters": {
                         "taxon_slug": "mugs",
-                        "attributeValues": [
+                        "attribute_values": [
```

Comment:
Ok, the problem is with our routeing definition... We should probably migrate it to camelCase also...